javascript/i18n/phonenumbers/demo.js

```diff
@@ -63,57 +64,68 @@ function phoneNumberParser() {
         case PNV.TOO_LONG:
           output.append('TOO_LONG');
           break;
+        case PNV.INVALID_LENGTH:
+          output.append('INVALID_LENGTH');
+          break;
       }
       // IS_POSSIBLE shouldn't happen, since we only call this if _not_
```

Comment:
Expand comment to cover IS_POSSIBLE_LOCAL_ONLY too

Reply:
Done.